core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -0,0 +1,323 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kafka.admin
+
+import java.util.Properties
+
+import joptsimple._
+import kafka.coordinator.ConsumerCoordinator
+import kafka.security.auth._
+import kafka.server.KafkaConfig
+import kafka.utils._
+import org.apache.kafka.common.security.auth.KafkaPrincipal
+import org.apache.kafka.common.utils.Utils
+
+object AclCommand {
+
+  val delimiter = ","
+  val nl = scala.util.Properties.lineSeparator
+
+  def main(args: Array[String]) {
+
+    val opts = new AclCommandOptions(args)
+
+    if (opts.options.has(opts.helpOpt))
+      CommandLineUtils.printUsageAndDie(opts.parser, "Usage:")
+
+    opts.checkArgs()
+
+    val authorizerProperties = new Properties()
+    if (opts.options.has(opts.authorizerPropertiesOpt)) {
+      val props = opts.options.valueOf(opts.authorizerPropertiesOpt).toString.split(delimiter).map(_.split("="))
+      props.foreach(pair => authorizerProperties.put(pair(0).trim, pair(1).trim))
```

Comment:
jopts allows you to specify multiple values for the same option. You just need to parse the option using options.valuesOf(), instead of options.valueOf() (there are a few usage of option.valuesOf() in ConfigCommand). Then, you can do the following from command line:

--authorization-property prop1=val1 --authorization-property prop2=val2 

We should change all other multiple value options (e.g. topic, operation, etc) accordingly and get rid of occurrences of split(delimiter).